File: 1920-build-array-from-permutation/1920-build-array-from-permutation.ts
function buildArray(nums: number[]): number[] {
    const n: number = nums.length;
    let result_array: number[] = [];
    for (let i = 0; i < n; i++) {
        result_array[i] = nums[nums[i]];
    }
    return result_array;
};

File: 1920-build-array-from-permutation/1920-build-array-from-permutation.cs
public class Solution {
    public int[] BuildArray(int[] nums) {
        int n = nums.Length ; 
        int[] resultArray = new int[n]; 
        for(int i=0 ;i<n ;i++){
            resultArray[i] = nums[nums[i]];
        }
        return resultArray;
    }
}